ingest/annotations.py

```diff
@@ -98,6 +96,14 @@ def create_data_frame(self):
         self.file = self.open_file(
             self.file_path, open_as='dataframe', dtype=dtypes, names=index, skiprows=2
         )[0]
+        # dtype of object allows mixed dtypes in columns, including numeric dtypes
+        # coerce group annotations pandas detects as non-object types to type string
```

Comment:
I had to re-read this line a few times to understand.  A tiny tweak would help readability.

```suggestion
        # coerce group annotations that pandas detects as non-object types to type string
```